closed/src/java.base/share/native/libjncrypto/NativeCrypto.c

```diff
@@ -61,7 +59,6 @@ static void handleErrors(void) {
         printf("Generating error message\n" );
         printf("%s\n", err);
     }
-    abort();
```

Comment:
This should probably use `ERR_error_string_n` instead of `ERR_error_string` to avoid sharing the static buffer by multiple threads.